fireant/slicer/widgets/reacttable.py

```diff
@@ -163,6 +164,41 @@ def map_display_values(df, dimensions):
 
         return dimension_display_values
 
+    @staticmethod
+    def map_hyperlink_templates(df, dimensions):
+        """
+        WRITEME
```

Comment:
Could you please add this header comment now rather than later.

Reply:
yes